src/lib/dialog/dialog-container.ts

```diff
@@ -46,8 +47,13 @@ export class MdDialogContainer extends BasePortalHost implements OnDestroy {
   /** Reference to the open dialog. */
   dialogRef: MdDialogRef<any>;
 
-  constructor(private _ngZone: NgZone) {
+  /** Exposes ViewContainerRef */
+  containerRef: ViewContainerRef;
+
+  constructor(private _viewContainerRef: ViewContainerRef, private _ngZone: NgZone) {
     super();
+
+    this.containerRef = this._viewContainerRef;
```

Comment:
had to do this, when I did just 

```
constructor(viewContainerRef: ViewContainerRef, private _ngZone: NgZone) {}
```

this https://github.com/fxck/material2/blob/9a0a28be6b551302363577462865fac760ad09a3/src/lib/dialog/dialog.ts#L212 was complaining that `viewContainerRef` doesn't exist